homework/app.js

```diff
@@ -0,0 +1,23 @@
+/* eslint-disable no-console */
```

Comment:
This file is probably an old version of your homework? You can remove it; or at least either name it that way (e.g. old_homework.js), or mention it in the comments at the top of the file ;). In general: always try to think which questions someone reading your code might ask him/herself; and answer them - ideally by clear naming; otherwise, by a comment.